SharpPcap/LibPcap/PcapHeader.cs

```diff
@@ -105,21 +105,24 @@ public unsafe static PcapHeader FromPointer(IntPtr pcap_pkthdr)
         /// <param name="captureLength">The length of the capture</param>
         public PcapHeader(uint seconds, uint microseconds, uint packetLength, uint captureLength)
         {
-            this.Seconds = seconds;
-            this.MicroSeconds = microseconds;
+            this.seconds = seconds;
+            this.microSeconds = microseconds;
             this.PacketLength = packetLength;
             this.CaptureLength = captureLength;
         }
 
         /// <summary>
         /// The seconds value of the packet's timestamp
         /// </summary>
-        public uint Seconds;
+        private uint seconds;
```

Comment:
Are you sure changing this to private won't break marshaling?

Reply:
I had exactly the same thought but all of the tests are passing... I can switch it back if we swap to a Timeval on ICaptureHeader as it won't overlap in terms of fields.